avm/res/cache/redis-enterprise/tests/e2e/amr/main.test.bicep

```diff
@@ -0,0 +1,50 @@
+targetScope = 'subscription'
+
+metadata name = 'Azure Managed Redis (Preview)'
```

Comment:
Please add the role assignment tests we have in other modules here as well, given that the module does offer the feature.